test/unit/controller/jobRoleControllerTest.ts

```diff
@@ -4,13 +4,23 @@ import express from 'express';
 import sinon from 'sinon';
```

Comment:
I'm not sure what I feel about these unit tests as I wouldn't personally use them to test the output HTML. I would just test that the correct methods were called (res.render, res.redirect) with the expected parameters.

Reply:
I guess I can look into that, but I'll change the job roles stuff in a separate PR.